app/pb_kits/playbook/pb_date_range_stacked/date_range_stacked.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbDateRangeStacked
+    class DateRangeStacked
+      include Playbook::Props
+
+      partial "pb_date_range_stacked/date_range_stacked"
+
+      prop :dark, type: Playbook::Props::Boolean, default: false
+      prop :end_date, type: Playbook::Props::Date, required: true
+      prop :start_date, type: Playbook::Props::Date, required: true
```

Comment:
@thestephenmarshall, I have made both of these as required fields but date field in Date Yeat Stacked Kit (rails) is not required. And for React how do I make [both of these](https://github.com/powerhome/playbook/pull/559/files#diff-4ce97b476aff57b44522d6539f5a80e4R18-R20) required if I wanna keep these kits consistent?

Reply:
To make a field required (as of now) you would just notate it using Flow syntax:

```jsx
endDate: Date, //required
startDate?: Date, //optional
```

Is that what you mean?

Reply:
If that's the case, then both rails and react APIs are consistent.